packages/drivers/odsp-driver/src/odspFluidFileLink.ts

```diff
@@ -38,7 +39,7 @@ export function encodeOdspFluidDataStoreLocator(locator: OdspFluidDataStoreLocat
             encodeURIComponent(locator.containerPackageName)}`;
     }
 
-    return btoa(locatorSerialized);
```

Comment:
Are these equivalent operations? I do not remember the details, but when we looked at it long time ago, btoa was not safe to convert strings. I wonder if
1. we should ban btoa across repo (assuming no for q below)
2. if we are introducing some changes to behavior that will break things.

Reply:
1.) btoa converts to base64 so I am just using an equivalent api which can be used in node too. This was the only place we are using it so I removed it.
2.) Also at the other end, I replaced atob with its equivalent. Also added test for the same. So we are not breaking anything.